index.html

```diff
@@ -255,10 +255,43 @@
                     button.appendChild(titleName);
                 });
 
+                editor.addAction('deleteBlock', function(editor, cell) {
```

Comment:
Could you please provide documentation, as a comment just above this block so as to what you've done.

Also, please provide comments within the code, explaining the 'important' lines. 

Thanks!